examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -21,11 +21,12 @@
      "name": "stdout",
```

Comment:
<p>This paragraph on name clashes should be deleted. Now variables in the posterior are available as <code>trace.posterior["var_name"]</code>  and sample stats as <code>trace.sample_stats["stats_name"]</code>,  there is no <code>trace["var_name"]</code>  option.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>